src/yunohost/log.py

```diff
@@ -371,6 +372,14 @@ def func_wrapper(*args, **kwargs):
             for field in exclude:
                 if field in context:
                     context.pop(field, None)
+
+            # Manage file or stream
```

Comment:
```suggestion
            # Context is made from args given to main function by argparse
            # This context will be added in extra parameters in yml file, so this context should 
            # be serializable and short enough (it will be displayed in webadmin)
            # Argparse can provide some File or Stream, so here we display the filename or
            # the IOBase, if we have no name.
```